### PR TITLE
add cross_language_framework_based_on_gramine document

### PR DESCRIPTION
**Pull request by hhr293**
add cross_language_framework_based_on_gramine document

### CODE REVIEW

File: documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ the computing resources of the remote SGX-enabled server and combine the
 data of the two parties to jointly run a specific program. The specific
 process is as follows.
 
-![]("../img/cross_language.jpeg")
+![image](https://github.com/hhr293/confidential-computing-zoo/blob/main/documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/img/cross_language.jpeg)
```
Comment (liangintel) [inline documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md:31]: should not be an absolute path

**Review by liangintel — CHANGES_REQUESTED** [review]
1. change Readme.md to index.md
2. Need to add this file into [documents/readthedoc/docs/source/index.rst]

File: documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+illustrate this process.
+
+User A and user B each have some private data locally. They want to use
+the computing resources of the remote SGX-enabled server and combine the
+data of the two parties to jointly run a specific program. The specific
+process is as follows.
+
+![image](https://github.com/hhr293/confidential-computing-zoo/blob/main/documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/img/cross_language.jpeg)
+
+1.  The SGX-enabled remote server will load the program into the enclave
```
Comment (yuanwu2017) [inline documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md:33]: Should align with the Image? sgx-enabled machine.

**Review by RodgerZhu — COMMENTED** [review]
Please submit PR to branch-dev/cross-lang-framework first.

File: documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    created by SGX and then generate a signature. The signature contains
```
Comment (yuanwu2017) [inline documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md:34]: What is signature? mr_enclave or mr_signer?

File: documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    the hash value of the program (to ensure that the program has not
+    been tampered with) and the Intel SGX certification certificate
+    (X.509 certificate chain to ensure that the machine is SGX enabled),
+    so programs placed in the enclave cannot be exposed or stolen);
+
+2.  The remote server sends the signature to both users A and B;
```
Comment (yuanwu2017) [inline documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md:40]: Is it a ratls procedure? or only send the signature?

File: documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+3.  After A and B receive the signature sent by the remote server, they
+    will verify the signature. After the verification is passed, they
+    will generate a key, then use the generated key to encrypt the local
+    data file, and pass the key and encrypted data file will be sent to
```
Comment (yuanwu2017) [inline documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md:45]: delete the "pass" or "will be sent"

**Comment by liangintel** [comment]
resubmit a new PR to dev branch

File: documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    the server through the secure channel;
+
+4.  After the remote server receives the key and encrypted data sent by
+    A and B, it will decrypt the data file in the SGX enclave, and then
+    call the CC Lib provided to execute the program ;
+
+5.  After the remote server finish the program and gets the result in
+    SGX enclave, it uses the keys get from the user to encrypt the
+    running result and send it back to users A and B.
```
Comment (yuanwu2017) [inline documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md:54]: It seems that it protects the transmission security?

**Pull request closed without merging**